scripts/bump-spin-formula.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+VERSION=$1
+FORMULA=${2:-Formula/Spin.rb}
+
+usage() {
+    echo "Usage: $0 <VERSION> [<FORMULA_PATH>]"
+    echo "Updates the Spin Formula for the specified release version"
+    echo "Example: $0 3.0.0"
+}
+
+if [ $# -ne 1 ]; then
+    usage
+    exit 1
+fi
+
+# Get the checksum file for the release
+wget -qO checksums.txt "https://github.com/fermyon/spin/releases/download/$VERSION/checksums-$VERSION.txt"
+
+# Update the formula version
```

Comment:
If we keep this technique of chopping off the leading character (assumed to be 'v') can we add a note here mentioning that is what we're doing below?  Bonus points if the logic can be improved to only chop off the leading character _if_ the character is actually a `v`.

Reply:
I add a check at the beginning of the script now to ensure it is prefixed.